rest-api/tools/import_organizations.py

```diff
@@ -176,38 +178,47 @@ def _entity_from_row(self, row):
                 zipCode=zip_code,
                 latitude=latitude,
                 longitude=longitude,
+                timeZoneId=time_zone_id,
                 phoneNumber=phone,
                 adminEmails=admin_email_addresses,
                 link=link)
 
   def _get_lat_long_for_site(self, address_1, city, state):
-    full_address = address_1 + ' ' +  city + ' ' + state
+    self.full_address = address_1 + ' ' +  city + ' ' + state
     try:
-      api_key = os.environ.get('API_KEY')
-      gmaps = googlemaps.Client(key=api_key)
-      geocode_result = gmaps.geocode(address_1 + '' +  city + ' ' +  state)[0]
+      self.api_key = os.environ.get('API_KEY')
```

Comment:
This doesn't get used outside this function I think, so technically you don't need to set it to a member var.

Reply:
Tested. It is required for timezone api.